packages/cli/src/build/config.ts

```diff
@@ -207,5 +221,38 @@ export function getRollupConfigs(pkg: Package, aliases: Aliases) {
     });
   }
 
+  if (hasWorkerField) {
+    configs.push({
+      config: getRollupConfig(
+        pkg,
+        pkg.entrypoints,
+        aliases,
+        "worker",
+        () => {}
+      ),
+      outputs: [
+        {
+          format: "cjs" as const,
+          entryFileNames: "[name].worker.cjs.js",
+          chunkFileNames: "dist/[name]-[hash].worker.cjs.js",
```

Comment:
q: is a CJS file even needed for a worker target? Can't we assume that a consumer is able to understand ESM?

Reply:
The same argument could be made for browser target so I added this to be consistent. IMO, this is more about bundler preference than the target's preference. 

I'm happy to take it out though if you feel like it is not needed.

Reply:
I agree with your sentiment - it's just that the browser field has been created in the past, where the tooling's state was different. I assume that for the newly introduced `exports` we can start from a clean slate - I don't feel super strong about it though, my bundlers should just never pick up CJS files for those and that's it. I just wonder if we can skip generating those for perf reasons and to spare some disk space etc.

Let's wait for @mitchellhamilton's opinion

Reply:
I feel like I agree with not having CommonJS builds on the worker condition with the caveat that we do this:
```json
"worker": {
  "module": "./dist/pkg.worker.esm.js"
}
```

I don't really want to put it the ESM one as the default since it's ESM aimed at bundlers, thoughts?